app/src/components/UI/Pages/VendorAppForm.tsx

```diff
@@ -1,58 +1,112 @@
-import React, { useEffect, useState } from "react";
+import React from "react";
 import { Form, Container, Button } from "semantic-ui-react";
 import Buttons from "../Atoms/Button/Buttons";
 import HeaderBar from "../Molecules/HeaderBar/HeaderBar";
-import styles from "./signup.module.css";
+import { Dropdown } from "semantic-ui-react";
+import styles from "./vendorappform.module.css";
 
 export default function VendorAppForm(): React.ReactElement {
   const storeUserInfo = () => {};
 
+  const timeOptionsFromValues = [
+    //options for business hours starting from...
+    "8:30 AM",
+    "9:00 AM",
+    "9:30 AM",
+    "10:00 AM",
+    "10:30 AM",
+    "11:00 AM",
+    "11:30 AM",
+    "12:00 AM",
+  ];
+
+  const timeOptionsToValues = [
+    //options for business hours starting to..
+    "8:30 PM",
+    "9:00 PM",
+    "9:30 PM",
+    "10:00 PM",
+    "10:30 PM",
+    "11:00 PM",
+    "11:30 PM",
+    "12:00 PM",
+  ];
+
+  const startTimeOptions = timeOptionsFromValues.map((element, key) => {
+    return {
+      key: timeOptionsFromValues[key],
+      text: element,
+      value: element,
+    };
+  });
+
+  const endTimeOptions = timeOptionsToValues.map((element, key) => {
+    return {
+      key: timeOptionsToValues[key],
+      text: element,
+      value: element,
+    };
+  });
+
   return (
     <Container className={styles.signUpWrapper}>
       <HeaderBar signUp={false} />
-      <h1>Sign Up Form (user account)</h1>
+      <h1>Sign Up Form (Vendor account)</h1>
       <Form className={styles.form}>
         <Form.Input
           // error={{ content: 'Please enter your first name', pointing: 'below' }}
           fluid
-          label="Address"
-          placeholder="Address"
+          label="Name"
+          placeholder="Name"
           id="form-input-first-name"
           required
           width={5}
         />
         <Form.Input
           // error='Please enter your last name'
           fluid
-          label=""
-          placeholder=""
+          label="Business Address"
+          placeholder="Business Address"
           required
           width={5}
         />
         <Form.Input
           // error='Please enter your last name'
           fluid
-          label="Phone"
-          placeholder="Phone"
+          label="Website URL"
+          placeholder="Website URL"
           required
           width={5}
         />
-
+        <h5>Business Hours: </h5>
+        <Dropdown
```

Comment:
I don't think vendors should specify their hours from the sign-up page. They should add it later, as well as the logo. A sign-up page should only ask for necessary information for signing up .

Reply:
I personally think there could be some vendors who have an idea of their business hours, and it would be better if that was specified as soon as possible. I think I can add a disclaimer saying that you can change that in the settings later on. But clarifying the business hours as early as possible is important, I believe.

Reply:
I will approve it. The business hours input should be changed in the future so that different hours for different weekdays can be inputted.